### PR TITLE
Settings Community informations

### PR DESCRIPTION
**Pull request by shiipou**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# ✨ Contributing to LWS ✨
```
Comment (alexesprit) [inline CONTRIBUTING.md]: Should be changed to "Contributing to setup-vlang-action" or something similar.

File: .github/.github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+A clear and concise description of what you expected to happen.
+
+**Screenshots and/or logs**
+If applicable, add screenshots and/or logs to help explain your problem.
+
+**Environment (please complete the following information):**
+ - Device: [e.g. Intel i9-9900k 64bit, 32Go of RAM, 128Go Free on 256Go SSD, ...]
+ - OS: [e.g. Manjaro XFCE 19.0.2, MacOS Catalina 10.15.4]
+ - Browser [e.g. chrome 82, safari 13.1]
+ - LWS version [e.g. 1.0.42]
```
Comment (alexesprit) [inline .github/.github/ISSUE_TEMPLATE/bug_report.md]: Should be changed to "Action version" or something similar.

File: .github/.github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+3. That throw an error in the console
+
+**Screenshots and/or logs**
+If applicable, add screenshots to help explain your problem.
+
+**Environment (please complete the following information):**
+ - Device: [e.g. Intel i9-9900k 64bit, 32Go of RAM, 128Go Free on 256Go SSD, ...]
+ - OS: [e.g. Manjaro XFCE 19.0.2, MacOS Catalina 10.15.4]
+ - Browser [e.g. chrome 82, safari 13.1]
+ - LWS version [e.g. 1.0.42]
```
Comment (alexesprit) [inline .github/.github/ISSUE_TEMPLATE/question.md]: Should be changed to "Action version" or something similar.

File: CONTRIBUTING.md (hunk 1 above)
Comment (shiipou) [reply CONTRIBUTING.md]: Haha copy past mistake XD
Thanks

**Comment by shiipou** [comment]
> We can also use shared documents (e.g. Code of Conduct, issue templates) for organization repositories: https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file.
> 
> If we go with this way, we can make issue templates to be more generic:
> 
> ```lisp
> **Environment (please complete the following information):**
>  - Version [e.g. 1.0.42] 
>  - Device: [e.g. Intel i9-9900k 64bit, 32Go of RAM, 128Go Free on 256Go SSD, ...]
>  - OS: [e.g. Manjaro XFCE 19.0.2, MacOS Catalina 10.15.4]
>  - Browser [e.g. chrome 82, safari 13.1]
> ```
> 
> I placed the version to the top of the list intentionally to avoid confusing with the browser version.
> 
> Aside my comments the PR looks fine.

Ok, I look how to make it.

Ok, I change this.

**Pull request merged**